guides/image_classification_with_vision_transformers.md

```diff
@@ -0,0 +1,55 @@
+# Image Classification with Vision Transformers
+
+related_spaces: abidlabs/vision-transformer
```

Comment:
include full link and more similar spaces. Can be those using transformers not necessarily vision.